packages/prisma/migrations/20240117062434_move_org_metadata_to_org_settings/migration.sql

```diff
@@ -0,0 +1,46 @@
+-- AlterTable
+ALTER TABLE "Team" ADD COLUMN     "isOrganization" BOOLEAN NOT NULL DEFAULT false;
+
+-- CreateTable
+CREATE TABLE "OrganizationSettings" (
+    "id" SERIAL NOT NULL,
+    "organizationId" INTEGER NOT NULL,
+    "isOrganizationConfigured" BOOLEAN NOT NULL DEFAULT false,
+    "isOrganizationVerified" BOOLEAN NOT NULL DEFAULT false,
+    "orgAutoAcceptEmail" TEXT NOT NULL,
+
+    CONSTRAINT "OrganizationSettings_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateIndex
+CREATE UNIQUE INDEX "OrganizationSettings_organizationId_key" ON "OrganizationSettings"("organizationId");
+
```

Comment:
Notice the constraint is not there now https://github.com/calcom/cal.com/pull/12828/files#r1453879560